src/server/rdb_load.cc

```diff
@@ -2466,6 +2478,9 @@ void RdbLoader::LoadItemsBuffer(DbIndex db_ind, const ItemsBuf& ib) {
     if (!res.is_new) {
       LOG(WARNING) << "RDB has duplicated key '" << item->key << "' in DB " << db_ind;
     }
+
+    if (auto* ts = es->tiered_storage(); ts && ts->ShouldStash(res.it->second))
+      ts->Stash(db_cntx.db_index, item->key, &res.it->second);
```

Comment:
nit: Consider merging `ShouldStash` and `Stash` into `TryStash` or something

Reply:
Renamed